spec.txt

```diff
@@ -3658,8 +3649,9 @@ these constructions.  (A recipe is provided below in the section entitled
 ## Block quotes
 
 A [block quote marker](@)
-consists of 0-3 spaces of initial indent, plus (a) the character `>` together
-with a following space, or (b) a single character `>` not followed by a space.
+consists of 0-3 spaces of indentation, plus (a) the character `>` together
+with a following space of indentation, or (b) a single character `>` not
+followed by a space of indentation.
```

Comment:
Here I'd remove "of indentation".  It's not clear that this space is really "indentation."